modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/src/main/resources/site-initializer/ddm-templates/release-notes-highlights-search/ddm-template.ftl

```diff
@@ -241,14 +254,14 @@
 									relatedContentsTitle = relatedContents.title
 								/>
 
-								<div class="features-box">
-									<div class="features-container">
+								<div class="d-flex features-box">
+									<a class="features-container" href="#">
```

Comment:
instead of using '#', typically we use 'javascript:;'. This prevents the link from jumping to the top of the page